### ISSUE TITLE
Environment issues

### ISSUE THREAD

**Issue by minhnimble** [SeintApril/Survey#8]
It was interesting to see that you have an `UrlHelper` to manage all the information relating to the backend. Now as per our our API [documentation](https://github.com/nimblehq/nimble-survey-web/wiki/REST-API#ios), there are more environments that our API should be able to support, which are `Staging` and `Production`, and you only handled the `Production` environment so far. It would be great if you could update the project implementation to handle both environments and point to the correct base endpoint for each environment as well. What do you think?

**Comment by SeintApril** [comment]
Yes. I add <code>UrlHelper</code> , to handle easily between different environment just changing information in one file.

**Comment by minhnimble** [comment]
@SeintApril hmm sounds interesting, but it is still unclear for me on the actual steps that you will need to do to handle the configuration there. Maybe you can go ahead and implement the different environments support using proper gitflow then 😁

**Comment by SeintApril** [comment]
-  I will create 2 different configurations, 2 different schemes, <code>Production</code> and <code>Staging</code>, connect schemes with their configurations. 
-  After that, I will add a baseURL variable of a String type in *.plist files and fill it with URLs and  call it from the code.

**Comment by minhnimble** [comment]
Sounds good to me, let me know once you complete the implementation 👍
